odoo/addons/base/models/ir_actions_report.py

```diff
@@ -377,7 +384,8 @@ def _prepare_html(self, html, report_model=False):
                     'subst': False,
                     'body': Markup(lxml.html.tostring(node, encoding='unicode')),
                     'base_url': base_url,
-                    'report_xml_id' : self.xml_id
+                    'report_xml_id': self.xml_id,
+                    'debug': self.env.context.get("debug"),
```

Comment:
Why this line don't use `query["debug"]` or `debug` previously defined ?
You can have `query` and `env.context` with different debug value ?

Reply:
to make debug part of the url i need to reformulate it as `?debug=True` is not valid to qweb
In the data passed to qweb I don't need to do this.